### ISSUE TITLE
ILR misbehaving with one dimension vectors

### ISSUE THREAD

**Issue by ltoran** [biocore/scikit-bio#1393]
Hello,

I'm trying to use the `ilr` module under `skbio.stats.composition`.
I'm having a problem with one dimensional basis.

Here's my code
```python
from skbio.stats.composition import ilr
import numpy as np
table = np.array([[  1.        ,  10.        ],
                  [  1.14141414,   9.90909091],
                  [  1.28282828,   9.81818182],
                  [  1.42424242,   9.72727273],
                  [  1.56565657,   9.63636364]])
basis = np.array([[ 0.80442968,  0.19557032]])
ilr_coords = ilr(table, basis=basis)
```

It seems to work when I cast the basis to 2 dimensions.  Any thoughts on this?

**Comment by jsevo** [comment]
I am having difficulty understanding what the analysis of 2D data is
supposed to yield. What's your use case for the ILR here?

**Comment by mortonjt** [comment]
This problem is popping up when there is a single basis vector (its actually a typo)

```python
basis = np.array([ 0.80442968,  0.19557032])
```
There is a problem with the `_check_orthogonality` function.
Its an easy fix, putting in a PR right now.

**Comment by jairideout** [comment]
Fixed in #1396.
